addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -281,12 +331,21 @@ export const editorCommands = {
                     }
                 }
             }
+            // When multiple <p>text1</p><p>text2</p><p>text3</p><p>text3</p> only start
+            // and end were converting to <li>, and rest were pasted as <p> only, convert
+            // remaining <p> to <li> before pasting on list.
```

Comment:
Do you mean this:
```suggestion
            // When inserting multiple adjacent <p> into a list, only the first
            // and last were converted to <li>. The rest was simply inserted as
            // <p>. Now convert these as well.
```
?